### ISSUE TITLE
11/30 Commit

### ISSUE THREAD

**Issue by rfehre** [rfehre/netartfinal#4]
Hey I've been forgetting to push and also generally have not been doing a ton on this. Sincerely sorry about that. Idk if Stach explained my airport nightmare but things have been all fucky and I'm getting back into the swing now.

Currently all that is displayed, in this version, is an array of spheres representing a number pulled from the ProPublica Congress API. Specifically the number '115' which I get from calling the results[0].congress endpoint. Members of congress.

But that data is irrelevant! Really this is just showing that we've got three.js working with data pulled from an API. Which is good news and, depending on a lot of factors for what we're tryna do here, is kind of the hard part. I think?

The direction I assume we should go at this point is to keep that relationship kinda simple (pull # data from APIs, display it against other # data from an API, elevate it aesthetically/conceptually with other visuals or other data, could be text or location stuff, open to whatever). I've kinda just been trying to figure out the logistics and haven't put a ton of thought into the conceptual side.

If you have questions about what's going on in this code let me know. It's not commented well, but I'll try to go back and do a better job of that soon.

Also feel free to start commenting here if you already have ideas that we could start pursuing with this framework.

**Comment by eccchobunny** [comment]
Jack can u respond to the group text I sent a few days ago? We should get together tomorrow as a group and i think we need to go in next week to meet with nick as a group, have to let him know by tomorrow night when, as well. I got the three js running in class and did that on my fork, dont think it was pulled to the main . one. Im taking care of that part of the project tho, so u dont need to really do anything there unless u are just into it that much

**Comment by rfehre** [comment]
I didn't get a group text :/ Tomorrow I have to meet with a different group for a different project, so that probably doesn't work. In general doing this as a group project was a probably just a bad idea lol. Maybe we could now all take this foundation and use it to represent different data as individual projects or something? Idk.

**Comment by eccchobunny** [comment]
uh ok

**Comment by rfehre** [comment]
well thats not v constructive. I mean i dunno what we're doing right now is all i mean. Seems like unless we've got some solid ideas on how to make this a group gig we should just take the repo we've got with data coming in and three.js hooked up and work with that. But we need like a solid idea of where to go with this that we can all work on individually, even if we do stick as a group, because meeting a ton in the next week is gonna be hard and might not make a ton of sense.

**Comment by eccchobunny** [comment]
What I mean is ok, I think we are just gonna work on our own, Collins n I, at this point, in that case and with the lack of communication you've given when there was more time and knowing the time frame. Sounds like you want to do that too anyway, wouldve been nice to get that clarification earlier when we were waiting for u to help solidify ideas. Its really kinda too tight at this point time wise. I messaged you wednesday about getting back to me by thursday about meeting up tomorrow, but I guess u didnt get that and also didnt reply even when I messaged u the same day outside the group text saying we needed to get together irl asap as well. We gonna work on our own at this point.

**Comment by rfehre** [comment]
Yeah I guess I don't really understand the status of the project right now, or where you guys are at with it, or what our project idea is. I'm not trying to suggest that I have or need to work alone, just that the bones of the project seem to be here already. And it might be easier for us to all chose our own data/APIs that we want to work with and visualize them in our own ways. Unless you guys have already gotten started on something that I was gone for, I just dunno if there's a thing yet.

But if we haven't liked moved on an idea yet, that seems more straightforward to me than all of us meeting up and then trying to navigate the git of it all over the next couple weeks.

And yeah, reiterating again, never got that group text. Just the one that you sent from class.

**Comment by dirtylittledirtbike** [comment]
No offense but of course you don't know the status of the project or where we're at with it because you've been unreachable and haven't been in communication with us...  If you wanna go your own direction or prefer to work alone thats fine just say, but this indirect communication and expression of a last minute desire to work alone is not productive and directly impacts me and Stach. If you're not going to be able to meet/communicate with us in the next week that will make this project difficult for us to execute, so if that's the case it's probably best we continue without you.

**Comment by rfehre** [comment]
No offense taken, definitely understand yr perspective. I don't know if there's been a bunch of communication that I've missed out on, but we haven't been talking thru github, and I received one text from Stach on Wednesday which I responded to with an explanation of why I was gone for the week (stuck in airport hell). I still don't know what our conceptual basis is or where you guys are at with it or what we're all working on as individuals.

I'm not trying to, like, sneakily leave the group. I was trying to get a read on where we're at & propose, what I thought, would be easier. At this point tho, yeah, I'm just gonna take a step back. Missing this week of classes has kind of gummed everything up for me anyway, and I don't mean to inflict my bullshit on you guys.

This repo is working using socket.io to bring in API data if you want to use it. If you have questions about it let me know.
